libraries/AP_HAL_ChibiOS/hwdef/RADIX2HD/pack_firmware.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash
+
+# Example script to pack firmware for RADIX 2 HD
+# Get the firmware packer tool from here: https://github.com/BrainFPV/brainfpv_fw_packer
+
+# Edit these paths to be correct
+PACKER=/shared/brainfpv/brainfpv_code/brainfpv_fw_packer/brainfpv_fw_packer/brainfpv_fw_packer.py
+PATH=/shared/brainfpv/code/ardupilot/tools/gcc-arm-none-eabi-6-2017-q2-update/bin/:${PATH}
```

Comment:
what is this script for? it seems to have personal paths? is it needed?

Reply:
The target uses a proprietary bootloader and needs a firmware file that is in a custom file format. The file can be created using the [BrainFPV Firmare  Packer](https://github.com/BrainFPV/brainfpv_fw_packer) script, which is available open source.

I fixed the `pack_firmware.sh` script such that it no longer uses personal paths.